po/fr.po

```diff
@@ -3148,6 +3160,24 @@ msgstr "Taux d'échantillonage:"
 msgid "Samples per buffer:"
 msgstr "Echantillons par buffer:"
 
+msgid "Sound &ports"
+msgstr "Ports audio"
+
+msgid "&Mapping output"
+msgstr "&Mappage des sorties"
+
+msgid "&Midi properties"
+msgstr "Propriétés MIDI"
+
+msgid "Auto add new devices"
+msgstr "Ajouter les pereferiques nouveux"
```

Comment:
'Ajouter automatiquement les nouveaux périphériques'

Reply:
Too long. It wont't fit.

Reply:
@rousseldenis @oleg68 
 Maybe then "Ajout auto de nouveaux appareils" ?

Reply:
Your variant is still fitting
![midi-fr](https://user-images.githubusercontent.com/11586506/151412619-f28b3991-10a0-4243-bc86-7f73d88c4d88.png)

The dialog size is limited by another very long label on the Options page: ``Gérer automatiquement le cache de l'orgue lors du chargement``

![Options-fr](https://user-images.githubusercontent.com/11586506/151413012-dc1d6354-3284-4995-9c64-462a413dd8c9.png)

May be we remove ``lors du chargement``? What is your opinion?

Reply:
@oleg68 I think it is better to remove "de l'orgue" instead of "lors du chargement" ... i think we can understand that it is the organ cache and not potatoes cache 🥳 

Then "Gérer automatiquement le cache lors du chargement"

As this option is in "Cache" part, maybe we can just put
"Gérer automatiquement lors du chargement"

@rousseldenis What do you think ?

Reply:
> Your variant is still fitting
> 
> ![midi-fr](https://user-images.githubusercontent.com/11586506/151412619-f28b3991-10a0-4243-bc86-7f73d88c4d88.png)
> 
> 

Or less longer 
"Ajout automatique des nouveaux périphériques"
?

Reply:
Thank you, I've applied your suggestions.

As shorter as better.